frontend/src/scenes/experiments/utils.test.ts

```diff
@@ -0,0 +1,82 @@
+import { EntityType, InsightType } from '~/types'
+
+import { getMinimumDetectableEffect } from './utils'
+
+describe('utils', () => {
+    it('Funnel experiment returns correct MDE', async () => {
+        const experimentInsightType = InsightType.FUNNELS
+        const trendResults = [
+            {
+                action: {
+                    id: '$pageview',
+                    type: 'events' as EntityType,
+                    order: 0,
+                    name: '$pageview',
+                    custom_name: null,
+                    math: 'total',
+                    math_group_type_index: null,
+                },
+                aggregated_value: 0,
+                label: '$pageview',
+                count: 0,
+                data: [],
+                labels: [],
+                days: [],
+            },
+        ]
+
+        let conversionMetrics = { averageTime: 0, stepRate: 0, totalRate: 0.01 }
+        expect(getMinimumDetectableEffect(experimentInsightType, conversionMetrics, trendResults)).toEqual(1)
+
+        conversionMetrics = { averageTime: 0, stepRate: 0, totalRate: 0.1 }
+        expect(getMinimumDetectableEffect(experimentInsightType, conversionMetrics, trendResults)).toEqual(5)
+
+        conversionMetrics = { averageTime: 0, stepRate: 0, totalRate: 0.3 }
+        expect(getMinimumDetectableEffect(experimentInsightType, conversionMetrics, trendResults)).toEqual(3)
+
+        conversionMetrics = { averageTime: 0, stepRate: 0, totalRate: 0.5 }
+        expect(getMinimumDetectableEffect(experimentInsightType, conversionMetrics, trendResults)).toEqual(5)
+
+        conversionMetrics = { averageTime: 0, stepRate: 0, totalRate: 0.8 }
```

Comment:
Can you also add another one for 99.9% / 100% ? Should be 1 as well

Reply:
Ah, maybe I'm misunderstanding how this works.

I thought the distance from 50% determines the target conversion rate increase.

So for conversion rates of 40% and 60%, the rate increase should be the same = 0.1

But this will lead to different MDEs because:
40% * 1.1 = 44% => MDE 4%
60% * 1.1 = 66% => MDE 6%

But you're saying the MDEs should be the same for them?

Reply:
Oh, I'm looking at your notes now and it's clearly written there, my bad 🤦

Reply:
Fixed, sorry for the confusion. Will leave it open in case you want to have another look.